wof/examples/production_configs/wofpy.conf

```diff
@@ -15,5 +15,5 @@ setgid www-data
 env PATH=/home/ubuntu/miniconda/envs/wofpy/bin
 
 # change directory to where the uWSGI configuration file is located then execute uwsgi with the config file
-chdir /var/www/timeseries
-exec uwsgi --ini timeseries.ini
\ No newline at end of file
+chdir /var/www/wofpy
+exec uwsgi --ini wofpy.ini
```

Comment:
> Can you point out where that generalization was done? Your in-line comment only refers to a description, and I don't see anything obvious in the rest of the file that points to that generalization.

Basically rather than `timeseries` it is now just called `wofpy` and when a user start or stop the server, they would only need to type for example `systemctl start wofpy` etc.